account_credit_control/models/credit_control_run.py

```diff
@@ -209,7 +210,7 @@ def run_channel_action(self):
             comms = comm_obj._generate_comm_from_credit_lines(email_lines)
             comms._generate_emails()
         if letter_lines:
-            wiz = self.env['credit.control.printer'].create({
+            wiz = self.env['credit.control.printer'].create([{
```

Comment:
This doesn't benefit from batch create, so no need to change this.

Reply:
I have not changed all creates. Just the ones that their model have a _default_ batch create